packages/agreement-process/tsconfig.check.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "noEmit": true,
+    "outDir": "dist"
```

Comment:
Do we need `outDir` if we have `noEmit`? Same for other tsconfig files with `noEmit`